### DISCUSSION TITLE
Integration with Turbo - or something else - to have persisting element

### DISCUSSION THREAD

**Discussion by jeromelebleu** [django-commons/django-unicorn#361] [Q&A]
I am building a website with an audio player which must persist across page loads. It works fine with [Turbo](https://turbo.hotwired.dev/handbook/building#persisting-elements-across-page-loads), but I would like to enhance the player with Unicorn by turning it into a controller.

Thus, I am wondering if using Unicorn could be compatible with Turbo - or if you know a more appropriate library or solution to achieve this goal?

Thanks in advance - and nice work by the way!

**Comment by adamghill** [comment]
I've never tried to integrate Turbo with Django, but if you already have something working I'm not sure it's worth it. 🤷  However, I'd love to hear about how it goes if you try and/or look at an example code repo to help troubleshoot anything you might run into. 👍

**Reply by jeromelebleu** [reply]
Thanks for you answer! I make use of Unicorn to build an [audio player](https://forge.cliss21.org/cliss21/wagtail-webradio/src/branch/main/wagtail_webradio/components/player.py) which is used in the website of a radio and turned persistent thanks to Turbo. Unsurprisingly, it worked well indeed!
